examples/api/lib/material/input_chip/input_chip.1.dart

```diff
@@ -0,0 +1,367 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+const List<String> _pizzaToppings = <String>[
+  'Olives',
+  'Tomato',
+  'Cheese',
+  'Pepperoni',
+  'Bacon',
+  'Onion',
+  'Jalapeno',
+  'Mushrooms',
+  'Pineapple',
+];
+
+void main() => runApp(const EditableChipFieldApp());
+
+class EditableChipFieldApp extends StatelessWidget {
+  const EditableChipFieldApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(useMaterial3: true),
+      home: const EditableChipFieldExample(),
+    );
+  }
+}
+
+class EditableChipFieldExample extends StatefulWidget {
+  const EditableChipFieldExample({super.key});
+
+  @override
+  EditableChipFieldExampleState createState() =>
```

Comment:
When defining a function with `=>`, put the entire definition on one line. Here and elsewhere.